site/docs/pages/wallet/wallet.mdx

```diff
@@ -61,6 +62,147 @@ export function WalletComponents() {
 </WalletComponents>
 
 
+## Anatomy
+```tsx twoslash
+import {
+  ConnectWallet,
+  Wallet,
+  WalletDropdown,
+  WalletDropdownBasename,
+  WalletDropdownLink,
+  WalletDropdownFundLink,
+  WalletDropdownDisconnect,
+} from '@coinbase/onchainkit/wallet';
+import { Identity } from '@coinbase/onchainkit/identity';
+import { Avatar, Name, Address, EthBalance } from '@coinbase/onchainkit/identity';
+
+
+// ---cut-before---
+<Wallet >
+  <ConnectWallet />
+  <WalletDropdown >
+    <Identity address="0x838aD0EAE54F99F1926dA7C3b6bFbF617389B4D9" >
+      <Avatar />
+      <Name />
+      <Address />
+      <EthBalance />
+    </Identity>
+    <WalletDropdownBasename />
+    <WalletDropdownLink href="https://keys.coinbase.com">
+      Wallet
+    </WalletDropdownLink>
+    <WalletDropdownFundLink />
+    <WalletDropdownDisconnect />
+  </WalletDropdown>
+</Wallet>
+// ---cut-after---
+```
+
+## API Reference
+
+### Wallet
+Serves as the main container for all wallet-related components.
+
+<PropsTable data={[{
+  name: 'children',
+  typeSimple: 'React.ReactNode',
+description: 'Content of the `Wallet` component',
+}, ]} />
+
+### ConnectWallet
+Handles the wallet connection process. Place child components inside to customize the connect button appearance.
```

Comment:
we need to standardize on what we call the "child components".  in other docs we refer to them as subcomponents, and children below references "customized content"

Reply:
I am in support of standardizing around the following:
1. Components that we ship are _subcomponents_ (e.g. ConnectWallet, NFTMedia, etc)
2. children the user passes in are just `children`